imgts/entrypoint.sh

```diff
@@ -37,11 +37,69 @@ ERRIMGS=''
 # It's possible for multiple simultaneous label updates to clash
 CLASHMSG='Labels fingerprint either invalid or resource labels have changed'
 
+# In an effort to avoid unintentional deletion of release-branch VM images
+# this image-use context must be detected.  This function accepts a single
+# argument: the Cirrus-CI build ID.  It attempts to determine if that build
+# occured on a non-main branch, and if so will return zero.  Otherwise,
+# it will return non-zero for executions on behalf of all PRs or tags.
+is_release_branch_image(){
+    local buildId api query result fltrpfx branch tag
+    buildId=$1
+    api="https://api.github.com/graphql"
+    query="{
+        \"query\": \"query {
+            build(id: $buildId) {
+                branch
+                tag
+                pullRequest
+              }\"
+        }"
+
+    # It's possible for an image to be missing it's build ID label.
+    # For example, the first time imgts operates on a new image.
+    if ((${#buildId}<12)); then
+        warn 0 "Empty/invalid BuildId value found on image, ignoring: '$buildId'"
+        return 1
+    fi
+
+    fltrpfx=".data.build"
```

Comment:
s t rly ncssry t bbrvt tht? Random letter combinations are hard to read, easy to misspell, and cause suffering in readers. Could that be renamed to something descriptive?

Reply:
It made more sense in the context I copy-pasted it from.  Yeah, sure I can rename it.  Thanks for pointing it out.